requirements.txt

```diff
@@ -2,4 +2,7 @@ openai==1.3.0
 instructor==0.3.4
 deepdiff==6.7.1
 termcolor==2.3.0
-python-dotenv==1.0.0
\ No newline at end of file
+python-dotenv==1.0.0
+pytest~=7.4.3
```

Comment:
pytest is a dev dependency. Should be separated. I suggest using poetry for package and dependency management

Reply:
Hi I know its not perfect, could you possibly help by sending me the config that this needs or updating the branch?